### ISSUE TITLE
Error start peertube docker ARM v6.2.0

### ISSUE THREAD

**Issue by Kitanit** [Chocobozzz/PeerTube#6482]
### Additional information

![2024-07-16_12-53](https://github.com/user-attachments/assets/342769ab-117f-4861-b9d7-abc6c052201a)

**Comment by Chocobozzz** [comment]
What is the content of `docker-volume/config`?

**Comment by Kitanit** [comment]
> What is the content of `docker-volume/config`?

version: "3.3"

services:

  # You can comment this webserver section if you want to use another webserver/proxy or test PeerTube in local
  #  webserver:
  #    image: chocobozzz/peertube-webserver:latest
  #    # If you don't want to use the official image and build one from sources:
  #    # build:
  #    #   context: .
  #    #   dockerfile: Dockerfile.nginx
  #    env_file:
  #      - .env
  #    ports:
  #     - ":80"
  #       #     - "443:443"
  #    volumes:
  #      - type: bind
  #        # Switch sources if you downloaded the whole repository
  #        #source: ../../nginx/peertube
  #        source: ./docker-volume/nginx/peertube
  #        target: /etc/nginx/conf.d/peertube.template
  #      - assets:/var/www/peertube/peertube-latest/client/dist:ro
  #      - ./docker-volume/data:/var/www/peertube/storage
  #        #      - certbot-www:/var/www/certbot
  #        #      - ./docker-volume/certbot/conf:/etc/letsencrypt
  #    depends_on:
  #      - peertube
  #    restart: "always"

  # You can comment this certbot section if you want to use another webserver/proxy or test PeerTube in local
  #   certbot:
  #     container_name: certbot
  #     image: certbot/certbot
  #     volumes:
  #       - ./docker-volume/certbot/conf:/etc/letsencrypt
  #       - certbot-www:/var/www/certbot
  #     restart: unless-stopped
  #     entrypoint: /bin/sh -c "trap exit TERM; while :; do certbot renew --webroot -w /var/www/certbot; sleep 12h & wait $${!}; done;"
  #     depends_on:
  #       - webserver

  peertube:
    # If you don't want to use the official image and build one from sources:
    # build:
    #   context: .
    #   dockerfile: ./support/docker/production/Dockerfile.bookworm
    image: chocobozzz/peertube:production-bookworm
    # Use a static IP for this container because nginx does not handle proxy host change without reload
    # This container could be restarted on crash or until the postgresql database is ready for connection
    networks:
      default:
        ipv4_address: 182.23.0.42
#    env_file:
#      - .env

    ports:
     - "1935:1935" # Comment if you don't want to use the live feature
     - "9005:9005" # Uncomment if you use another webserver/proxy or test PeerTube in local, otherwise not suitable for production
    volumes:
      # Remove the following line if you want to use another webserver/proxy or test PeerTube in local
      # - assets:/app/client/dist
      - ./docker-volume/data:/data
      - ./docker-volume/config:/config
    depends_on:
      - postgres
      - redis
    restart: "always"

  postgres:
    image: postgres:13-alpine
 #   env_file:
 #     - .env
    volumes:
      - ./docker-volume/db:/var/lib/postgresql/data
    restart: "always"

  redis:
    image: redis:6-alpine
    volumes:
      - ./docker-volume/redis:/data
    restart: "always"

networks:
  default:
    ipam:
      driver: default
      config:
      - subnet: 182.23.0.0/16

**Comment by Chocobozzz** [comment]
I meant the content of your `docker-volume/config` on your disk, not the content of your `docker-compose.yml`

**Comment by Kitanit** [comment]
> I meant the content of your `docker-volume/config` on your disk, not the content of your `docker-compose.yml`
oops 

```
listen:
  hostname: '0.0.0.0'
  port: 9005

# Correspond to your reverse proxy server_name/listen configuration (i.e., your public PeerTube instance URL)
webserver:
  https: true
  hostname: 'peertube.in.ua'
  port: 443

# Secrets you need to generate the first time you run PeerTube
secrets:
  # Generate one using `openssl rand -hex 32`
  peertube: 'server'

rates_limit:
  api:
    # 50 attempts in 10 seconds
    window: 10 seconds
    max: 50
  login:
    # 15 attempts in 5 min
    window: 5 minutes
    max: 15
  signup:
    # 2 attempts in 5 min (only succeeded attempts are taken into account)
    window: 5 minutes
    max: 2
  ask_send_email:
    # 3 attempts in 5 min
    window: 5 minutes
    max: 3
  receive_client_log:
    # 10 attempts in 10 min
    window: 10 minutes
    max: 10

oauth2:
  token_lifetime:
    access_token: '1 day'
    refresh_token: '2 weeks'

# Proxies to trust to get real client IP
# If you run PeerTube just behind a local proxy (nginx), keep 'loopback'
# If you run PeerTube behind a remote proxy, add the proxy IP address (or subnet)
trust_proxy:
  - 'loopback'

# Your database name will be database.name OR 'peertube'+database.suffix
database:
  hostname: 'postgres'
  port: 5432
  ssl: false
  suffix: ''
  username: 'peertube'
  password: 'secret'
  pool:
    max: 5

# Redis server for short time storage
# You can also specify a 'socket' path to a unix socket but first need to
# set 'hostname' and 'port' to null
redis:
  hostname: 'redis'
  port: 6379
#  auth: blabla
  db: 0

# SMTP server to send emails
smtp:
  # smtp or sendmail
  transport: smtp
  # Path to sendmail command. Required if you use sendmail transport
  sendmail: null
  hostname: 'mail.securetown.top'
  port: 465 # If you use StartTLS: 587
  username: 'noreply@peertube.in.ua'
  password: 'secret'
  tls: true # If you use StartTLS: false
  disable_starttls: false
  ca_file: null # Used for self signed certificates
  from_address: 'noreply@peertube.in.ua'

email:
  body:
    signature: 'PeerTube'
  subject:
    prefix: '[PeerTube]'

# Update default PeerTube values
# Set by API when the field is not provided and put as default value in client
defaults:
  # Change default values when publishing a video (upload/import/go Live)
  publish:
    download_enabled: true

    comments_enabled: true

    # public = 1, unlisted = 2, private = 3, internal = 4
    privacy: 1

    # CC-BY = 1, CC-SA = 2, CC-ND = 3, CC-NC = 4, CC-NC-SA = 5, CC-NC-ND = 6, Public Domain = 7
    # You can also choose a custom licence value added by a plugin
    # No licence by default
    licence: null

  p2p:
    # Enable P2P by default in PeerTube client
    # Can be enabled/disabled by anonymous users and logged in users
    webapp:
      enabled: true

    # Enable P2P by default in PeerTube embed
    # Can be enabled/disabled by URL option
    embed:
      enabled: true

# From the project root directory
storage:
  tmp: '/data/storage/tmp/' # Use to download data (imports etc), store uploaded files before and during processing...
  tmp_persistent: '/data/storage/tmp-persistent/' # Use to download data (imports etc), store uploaded files before and during processing...
  bin: '/data/storage/bin/'
  original_video_files: '/data/storage/original_video_files'
  avatars: '/data/storage/avatars/'
  web_videos: '/data/storage/web-videos/'
  streaming_playlists: '/data/storage/streaming-playlists/'
  redundancy: '/data/storage/redundancy/'
  logs: '/data/storage/\logs/'
  previews: '/data/storage/previews/'
  thumbnails: '/data/storage/thumbnails/'
  torrents: '/data/storage/torrents/'
  captions: '/data/storage/captions/'
  cache: '/data/storage/cache/'
  plugins: '/data/storage/plugins/'
  well_known: '/data/storage/well-known/'
  storyboards: '/data/storage/storyboards/'
  # Overridable client files in client/dist/assets/images:
  # - logo.svg
  # - favicon.png
  # - default-playlist.jpg
  # - default-avatar-account.png
  # - default-avatar-video-channel.png
  # - and icons/*.png (PWA)
  # Could contain for example assets/images/favicon.png
  # If the file exists, peertube will serve it
  # If not, peertube will fallback to the default file
  client_overrides: '/data/storage/client-overrides/'

static_files:
  # Require and check user authentication when accessing private files (internal/private video files)
  private_files_require_auth: true

object_storage:
  enabled: true

  # Without protocol, will default to HTTPS
  endpoint: 's3.eu-central-003.backblazeb2.com' # 's3.amazonaws.com' or 's3.fr-par.scw.cloud' for example

  region: 'eu-central-003'

  upload_acl:
    # Set this ACL on each uploaded object of public/unlisted videos
    # Use null if your S3 provider does not support object ACL
    public: 'public-read'
    # Set this ACL on each uploaded object of private/internal videos
    # PeerTube can proxify requests to private objects so your users can access them
    # Use null if your S3 provider does not support object ACL
    private: null

  proxy:
    # If private files (private/internal video files) have a private ACL, users can't access directly the ressource
    # PeerTube can proxify requests between your object storage service and your users
    # If you disable PeerTube proxy, ensure you use your own proxy that is able to access the private files
    # Or you can also set a public ACL for private files in object storage if you don't want to use a proxy
    proxify_private_files: true

  credentials:
    # You can also use AWS_ACCESS_KEY_ID env variable
    access_key_id: 'secret'
    # You can also use AWS_SECRET_ACCESS_KEY env variable
    secret_access_key: 'secret'

  # Maximum amount to upload in one request to object storage
  max_upload_part: 100MB
  max_request_attempts: 5

  web_videos:
    bucket_name: 'Peertube-UAFediland'
    prefix: 'web-videos/'

  # Use the same bucket as for web videos but with a different prefix
  streaming_playlists:
    bucket_name: 'Peertube-UAFediland'
    prefix: 'streaming-playlists/'

log:
  level: 'warn' # 'debug' | 'info' | 'warn' | 'error'

  rotation:
    enabled : true # Enabled by default, if disabled make sure that 'storage.logs' is pointing to a folder handled by logrotate
    max_file_size: 12MB
    max_files: 20

  anonymize_ip: false

  log_ping_requests: true
  log_tracker_unknown_infohash: true

  prettify_sql: false

  # Accept warn/error logs coming from the client
  accept_client_log: true

# Support of Open Telemetry metrics and tracing
# For more information: https://docs.joinpeertube.org/maintain/observability
open_telemetry:
  metrics:
    enabled: false

    http_request_duration:
      # You can disable HTTP request duration metric that can have a high tag cardinality
      enabled: true

    # Create a prometheus exporter server on this port so prometheus server can scrape PeerTube metrics
    prometheus_exporter:
      hostname: '127.0.0.1'
      port: 9091

  tracing:
    enabled: false

    # Send traces to a Jaeger compatible endpoint
    jaeger_exporter:
      endpoint: ''

trending:
  videos:
    interval_days: 7 # Compute trending videos for the last x days for 'most-viewed' algorithm

    algorithms:
      enabled:
        - 'hot' # Adaptation of Reddit's 'Hot' algorithm
        - 'most-viewed' # Number of views in the last x days
        - 'most-liked' # Global views since the upload of the video

      default: 'most-viewed'

# Cache remote videos on your server, to help other instances to broadcast the video
# You can define multiple caches using different sizes/strategies
# Once you have defined your strategies, choose which instances you want to cache in admin -> manage follows -> following
redundancy:
  videos:
    check_interval: '1 hour' # How often you want to check new videos to cache
    strategies: # Just uncomment strategies you want
#      -
#        size: '10GB'
#        # Minimum time the video must remain in the cache. Only accept values > 10 hours (to not overload remote instances)
#        min_lifetime: '48 hours'
#        strategy: 'most-views' # Cache videos that have the most views
#      -
#        size: '10GB'
#        # Minimum time the video must remain in the cache. Only accept values > 10 hours (to not overload remote instances)
#        min_lifetime: '48 hours'
#        strategy: 'trending' # Cache trending videos
#      -
#        size: '10GB'
#        # Minimum time the video must remain in the cache. Only accept values > 10 hours (to not overload remote instances)
#        min_lifetime: '48 hours'
#        strategy: 'recently-added' # Cache recently added videos
#        min_views: 10 # Having at least x views

# Other instances that duplicate your content
remote_redundancy:
  videos:
    # 'nobody': Do not accept remote redundancies
    # 'anybody': Accept remote redundancies from anybody
    # 'followings': Accept redundancies from instance followings
    accept_from: 'anybody'

csp:
  enabled: false
  report_only: true # CSP directives are still being tested, so disable the report only mode at your own risk!
  report_uri:

security:
  # Set the X-Frame-Options header to help to mitigate clickjacking attacks
  frameguard:
    enabled: true

  # Set x-powered-by HTTP header to "PeerTube"
  # Can help remote software to know this is a PeerTube instance
  powered_by_header:
    enabled: true

tracker:
  # If you disable the tracker, you disable the P2P on your PeerTube instance
  enabled: true
  # Only handle requests on your videos
  # If you set this to false it means you have a public tracker
  # Then, it is possible that clients overload your instance with external torrents
  private: true
  # Reject peers that do a lot of announces (could improve privacy of TCP/UDP peers)
  reject_too_many_announces: false

history:
  videos:
    # If you want to limit users videos history
    # -1 means there is no limitations
    # Other values could be '6 months' or '30 days' etc (PeerTube will periodically delete old entries from database)
    max_age: -1

views:
  videos:
    # PeerTube creates a database entry every hour for each video to track views over a period of time
    # This is used in particular by the Trending page
    # PeerTube could remove old remote video views if you want to reduce your database size (video view counter will not be altered)
    # -1 means no cleanup
    # Other values could be '6 months' or '30 days' etc (PeerTube will periodically delete old entries from database)
    remote:
      max_age: '30 days'

    # PeerTube buffers local video views before updating and federating the video
    local_buffer_update_interval: '30 minutes'

    ip_view_expiration: '1 hour'

# Used to get country location of views of local videos
geo_ip:
  enabled: true

  country:
    database_url: 'https://dbip.mirror.framasoft.org/files/dbip-country-lite-latest.mmdb'

plugins:
  # The website PeerTube will ask for available PeerTube plugins and themes
  # This is an unmoderated plugin index, so only install plugins/themes you trust
  index:
    enabled: true
    check_latest_versions_interval: '12 hours' # How often you want to check new plugins/themes versions
    url: 'https://packages.joinpeertube.org'

federation:
  videos:
    federate_unlisted: false

    # Add a weekly job that cleans up remote AP interactions on local videos (shares, rates and comments)
    # It removes objects that do not exist anymore, and potentially fix their URLs
    cleanup_remote_interactions: true

peertube:
  check_latest_version:
    # Check and notify admins of new PeerTube versions
    enabled: true
    # You can use a custom URL if your want, that respect the format behind https://joinpeertube.org/api/v1/versions.json
    url: 'https://joinpeertube.org/api/v1/versions.json'

webadmin:
  configuration:
    edition:
      # Set this to false if you don't want to allow config edition in the web interface by instance admins
      allowed: true

# XML, Atom or JSON feeds
feeds:
  videos:
    # Default number of videos displayed in feeds
    count: 20

  comments:
    # Default number of comments displayed in feeds
    count: 20

###############################################################################
#
# From this point, almost all following keys can be overridden by the web interface
# (local-production.json file). If you need to change some values, prefer to
# use the web interface because the configuration will be automatically
# reloaded without any need to restart PeerTube
#
# /!\ If you already have a local-production.json file, modification of some of
# the following keys will have no effect /!\
#
###############################################################################

cache:
  previews:
    size: 500 # Max number of previews you want to cache
  captions:
    size: 500 # Max number of video captions/subtitles you want to cache
  torrents:
    size: 500 # Max number of video torrents you want to cache

admin:
  # Used to generate the root user at first startup
  # And to receive emails from the contact form
  email: 'admin@m0e.space'

contact_form:
  enabled: true

signup:
  enabled: false

  limit: 10 # When the limit is reached, registrations are disabled. -1 == unlimited

  minimum_age: 16 # Used to configure the signup form

  # Users fill a form to register so moderators can accept/reject the registration
  requires_approval: true
  requires_email_verification: false

  filters:
    cidr: # You can specify CIDR ranges to whitelist (empty = no filtering) or blacklist
      whitelist: []
      blacklist: []

user:
  # Default value of maximum video bytes the user can upload (does not take into account transcoded files)
  # Byte format is supported ("1GB" etc)
  # -1 == unlimited
  video_quota: -1
  video_quota_daily: -1

video_channels:
  max_per_user: 20 # Allows each user to create up to 20 video channels.

# If enabled, the video will be transcoded to mp4 (x264) with `faststart` flag
# In addition, if some resolutions are enabled the mp4 video file will be transcoded to these new resolutions
# Please, do not disable transcoding since many uploaded videos will not work
transcoding:
  enabled: true

  # Allow your users to upload .mkv, .mov, .avi, .wmv, .flv, .f4v, .3g2, .3gp, .mts, m2ts, .mxf, .nut videos
  allow_additional_extensions: true

  # If a user uploads an audio file, PeerTube will create a video by merging the preview file and the audio file
  allow_audio_files: true

  # Amount of threads used by ffmpeg for 1 transcoding job
  threads: 3
  # Amount of transcoding jobs to execute in parallel
  concurrency: 1

  # Choose the transcoding profile
  # New profiles can be added by plugins
  # Available in core PeerTube: 'default'
  profile: 'default'

  resolutions: # Only created if the original video has a higher resolution, uses more storage!
    0p: false # audio-only (creates mp4 without video stream, always created when enabled)
    144p: false
    240p: false
    360p: false
    480p: false
    720p: true
    1080p: true
    1440p: false
    2160p: false

  # Transcode and keep original resolution, even if it's above your maximum enabled resolution
  always_transcode_original_resolution: true

  # Generate videos in a WebTorrent format (what we do since the first PeerTube release)
  # If you also enabled the hls format, it will multiply videos storage by 2
  # If disabled, breaks federation with PeerTube instances < 2.1
  web_videos:
    enabled: true

  # /!\ Requires ffmpeg >= 4.1
  # Generate HLS playlists and fragmented MP4 files. Better playback than with WebTorrent:
  #     * Resolution change is smoother
  #     * Faster playback in particular with long videos
  #     * More stable playback (less bugs/infinite loading)
  # If you also enabled the webtorrent format, it will multiply videos storage by 2
  hls:
    enabled: true

live:
  enabled: true

  # Limit lives duration
  # -1 == unlimited
  max_duration: -1 # For example: '5 hours'

  # Limit max number of live videos created on your instance
  # -1 == unlimited
  max_instance_lives: 20

  # Limit max number of live videos created by a user on your instance
  # -1 == unlimited
  max_user_lives: 3

  # Allow your users to save a replay of their live
  # PeerTube will transcode segments in a video file
  # If the user daily/total quota is reached, PeerTube will stop the live
  # /!\ transcoding.enabled (and not live.transcoding.enabled) has to be true to create a replay
  allow_replay: true

  # Allow your users to change latency settings (small latency/default/high latency)
  # Small latency live streams cannot use P2P
  # High latency live streams can increase P2P ratio
  latency_setting:
    enabled: true

  # Your firewall should accept traffic from this port in TCP if you enable live
  rtmp:
    enabled: true

    # Listening hostname/port for RTMP server
    # '::' to listen on IPv6 and IPv4, '0.0.0.0' to listen on IPv4
    # Use null to automatically listen on '::' if IPv6 is available, or '0.0.0.0' otherwise
    hostname: null
    port: 1935

    # Public hostname of your RTMP server
    # Use null to use the same value than `webserver.hostname`
    public_hostname: null

  rtmps:
    enabled: false

    # Listening hostname/port for RTMPS server
    # '::' to listen on IPv6 and IPv4, '0.0.0.0' to listen on IPv4
    # Use null to automatically listen on '::' if IPv6 is available, or '0.0.0.0' otherwise
    hostname: null
    port: 1936

    # Absolute paths
    key_file: ''
    cert_file: ''

    # Public hostname of your RTMPS server
    # Use null to use the same value than `webserver.hostname`
    public_hostname: null

  # Allow to transcode the live streaming in multiple live resolutions
  transcoding:
    enabled: true
    threads: 2

    # Choose the transcoding profile
    # New profiles can be added by plugins
    # Available in core PeerTube: 'default'
    profile: 'default'

    resolutions:
      144p: false
      240p: false
      360p: true
      480p: false
      720p: false
      1080p: false
      1440p: false
      2160p: false

    # Also transcode original resolution, even if it's above your maximum enabled resolution
    always_transcode_original_resolution: true

video_studio:
  # Enable video edition by users (cut, add intro/outro, add watermark etc)
  # If enabled, users can create transcoding tasks as they wish
  enabled: false

import:
  # Add ability for your users to import remote videos (from YouTube, torrent...)
  videos:
    # Amount of import jobs to execute in parallel
    concurrency: 1

    # Set a custom video import timeout to not block import queue
    timeout: '2 hours'

    # Classic HTTP or all sites supported by youtube-dl https://rg3.github.io/youtube-dl/supportedsites.html
    http:
      # We recommend to use a HTTP proxy if you enable HTTP import to prevent private URL access from this server
      # See https://docs.joinpeertube.org/maintain/configuration#security for more information
      enabled: false

      youtube_dl_release:
        # Direct download URL to youtube-dl binary
        # Github releases API is also supported
        # Examples:
        #   * https://api.github.com/repos/ytdl-org/youtube-dl/releases
        #   * https://api.github.com/repos/yt-dlp/yt-dlp/releases
        #   * https://yt-dl.org/downloads/latest/youtube-dl
        url: 'https://api.github.com/repos/yt-dlp/yt-dlp/releases'

        # Release binary name: 'yt-dlp' or 'youtube-dl'
        name: 'yt-dlp'

        # Path to the python binary to execute for youtube-dl or yt-dlp
        python_path: '/usr/bin/python3'

      # IPv6 is very strongly rate-limited on most sites supported by youtube-dl
      force_ipv4: false

    # Magnet URI or torrent file (use classic TCP/UDP/WebSeed to download the file)
    torrent:
      # We recommend to only enable magnet URI/torrent import if you trust your users
      # See https://docs.joinpeertube.org/maintain/configuration#security for more information
      enabled: false

  # Add ability for your users to synchronize their channels with external channels, playlists, etc.
  video_channel_synchronization:
    enabled: false

    max_per_user: 10

    check_interval: 1 hour

    # Number of latest published videos to check and to potentially import when syncing a channel
    videos_limit_per_synchronization: 10

    # Max number of videos to import when the user asks for full sync
    full_sync_videos_limit: 1000

auto_blacklist:
  # New videos automatically blacklisted so moderators can review before publishing
  videos:
    of_users:
      enabled: false

# Instance settings
instance:
  name: 'PeerTube'
  short_description: 'PeerTube, an ActivityPub-federated video streaming platform using P2P directly in your web browser.'
  description: 'Welcome to this PeerTube instance!' # Support markdown
  terms: 'No terms for now.' # Support markdown
  code_of_conduct: '' # Supports markdown

  # Who moderates the instance? What is the policy regarding NSFW videos? Political videos? etc
  moderation_information: '' # Supports markdown

  # Why did you create this instance?
  creation_reason: '' # Supports Markdown

  # Who is behind the instance? A single person? A non profit?
  administrator: '' # Supports Markdown

  # How long do you plan to maintain this instance?
  maintenance_lifetime: '' # Supports Markdown

  # How will you pay the PeerTube instance server? With your own funds? With users donations? Advertising?
  business_model: '' # Supports Markdown

  # If you want to explain on what type of hardware your PeerTube instance runs
  # Example: '2 vCore, 2GB RAM...'
  hardware_information: '' # Supports Markdown

  # What are the main languages of your instance? To interact with your users for example
  # Uncomment or add the languages you want
  # List of supported languages: https://peertube.cpy.re/api/v1/videos/languages
  languages:
#    - en
#    - es
#    - fr

  # You can specify the main categories of your instance (dedicated to music, gaming or politics etc)
  # Uncomment or add the category ids you want
  # List of supported categories: https://peertube.cpy.re/api/v1/videos/categories
  categories:
#    - 1  # Music
#    - 2  # Films
#    - 3  # Vehicles
#    - 4  # Art
#    - 5  # Sports
#    - 6  # Travels
#    - 7  # Gaming
#    - 8  # People
#    - 9  # Comedy
#    - 10 # Entertainment
#    - 11 # News & Politics
#    - 12 # How To
#    - 13 # Education
#    - 14 # Activism
#    - 15 # Science & Technology
#    - 16 # Animals
#    - 17 # Kids
#    - 18 # Food

  default_client_route: '/videos/trending'

  # Whether or not the instance is dedicated to NSFW content
  # Enabling it will allow other administrators to know that you are mainly federating sensitive content
  # Moreover, the NSFW checkbox on video upload will be automatically checked by default
  is_nsfw: false
  # By default, `do_not_list` or `blur` or `display` NSFW videos
  # Could be overridden per user with a setting
  default_nsfw_policy: 'do_not_list'

  customizations:
    javascript: '' # Directly your JavaScript code (without <script> tags). Will be eval at runtime
    css: '' # Directly your CSS code (without <style> tags). Will be injected at runtime
  # Robot.txt rules. To disallow robots to crawl your instance and disallow indexation of your site, add `/` to `Disallow:`
  robots: |
    User-agent: *
    Disallow:
  # /.well-known/security.txt rules. This endpoint is cached, so you may have to wait a few hours before viewing your changes
  # To discourage researchers from testing your instance and disable security.txt integration, set this to an empty string
  securitytxt:
    '# If you would like to report a security issue\n# you may report it to:\nContact: https://github.com/Chocobozzz/PeerTube/blob/develop/SECURITY.md\nContact: mailto:'

services:
  # Cards configuration to format video in Twitter
  twitter:
    username: '@Chocobozzz' # Indicates the Twitter account for the website or platform on which the content was published
    # If true, a video player will be embedded in the Twitter feed on PeerTube video share
    # If false, we use an image link card that will redirect on your PeerTube instance
    # Change it to `true`, and then test on https://cards-dev.twitter.com/validator to see if you are whitelisted
    whitelisted: false

followers:
  instance:
    # Allow or not other instances to follow yours
    enabled: true
    # Whether or not an administrator must manually validate a new follower
    manual_approval: false

followings:
  instance:
    # If you want to automatically follow back new instance followers
    # If this option is enabled, use the mute feature instead of deleting followings
    # /!\ Don't enable this if you don't have a reactive moderation team /!\
    auto_follow_back:
      enabled: false

    # If you want to automatically follow instances of the public index
    # If this option is enabled, use the mute feature instead of deleting followings
    # /!\ Don't enable this if you don't have a reactive moderation team /!\
    auto_follow_index:
      enabled: false
      # Host your own using https://framagit.org/framasoft/peertube/instances-peertube#peertube-auto-follow
      index_url: ''

theme:
  default: 'default'

broadcast_message:
  enabled: false
  message: '' # Support markdown
  level: 'info' # 'info' | 'warning' | 'error'
  dismissable: false

search:
  # Add ability to fetch remote videos/actors by their URI, that may not be federated with your instance
  # If enabled, the associated group will be able to "escape" from the instance follows
  # That means they will be able to follow channels, watch videos, list videos of non followed instances
  remote_uri:
    users: true
    anonymous: false

  # Use a third party index instead of your local index, only for search results
  # Useful to discover content outside of your instance
  # If you enable search_index, you must enable remote_uri search for users
  # If you do not enable remote_uri search for anonymous user, your instance will redirect the user on the origin instance
  # instead of loading the video locally
  search_index:
    enabled: false
    # URL of the search index, that should use the same search API and routes
    # than PeerTube: https://docs.joinpeertube.org/api/rest-reference.html
    # You should deploy your own with https://framagit.org/framasoft/peertube/search-index,
    # and can use https://search.joinpeertube.org/ for tests, but keep in mind the latter is an unmoderated search index
    url: ''
    # You can disable local search, so users only use the search index
    disable_local_search: false
    # If you did not disable local search, you can decide to use the search index by default
    is_default_search: false

# PeerTube client/interface configuration
client:
  videos:
    miniature:
      # By default PeerTube client displays author username
      prefer_author_display_name: false
      display_author_avatar: false
    resumable_upload:
      # Max size of upload chunks, e.g. '90MB'
      # If null, it will be calculated based on network speed
      max_chunk_size: null

  menu:
    login:
      # If you enable only one external auth plugin
      # You can automatically redirect your users on this external platform when they click on the login button
      redirect_on_single_external_auth: false


thumbnails:
  # When automatically generating a thumbnail from the video
  generation_from_video:
    # How many frames to analyze at the middle of the video to select the most appropriate one
    # Increasing this value will increase CPU and memory usage when generating the thumbnail, especially for high video resolution
    # Minimum value is 2
    frames_to_analyze: 20

  # Only two sizes are currently supported for now (not less, not more)
  # 1 size for the thumbnail (displayed in video miniatures)
  # 1 size for the preview (displayed in the video player)
  sizes:
      width: 850
      height: 480
``

**Comment by Chocobozzz** [comment]
`thumbnails.size` is invalid. It must be an array. Please check https://github.com/Chocobozzz/PeerTube/blob/develop/config/production.yaml.example#L479
